### DISCUSSION TITLE
Can't change the mount point's name because of Liquidsoap

### DISCUSSION THREAD

**Discussion by greatdevvie** [AzuraCast/AzuraCast#6331] [Q&A]
Good morning.
Is there any way to change the 'icy-name' of only a mount?

I tried to do it through Custom Frontend Configuration in Mount Points using http-headers, but I failed and the 'icy-name' is still standard.

Any other ways to do it?
Thank you for help.

**Comment by greatdevvie** [comment]
As a suggestion to @BusterNeece please make the Stream Name customizable in the Basic Info.

**Comment by gunsar** [comment]
Custom Frontend Configuration in Mount Points  : 

`<stream-name>............</stream-name>`
`<stream-description>......................</stream-description>`
`<stream-url>.......................</stream-url>`
`<genre>.....................</genre>`

`<http-headers>`
`.................`
`.................`
`</http-headers>`

**Reply by greatdevvie** [reply]
this method doesn't work because of liquidsoap that takes the stream name to its value instead of the <stream-name> value.
